From cbda79de8e4397d65f860d50a70c4eb9f91b28ce Mon Sep 17 00:00:00 2001
From: Yurii Rashkovskii <yrashk@gmail.com>
Date: Fri, 23 Feb 2018 17:36:28 +0700
Subject: [PATCH] Problem: progress bar is not very informative

It does show how far along it is but that's about it

Solution: show current item being imported
---
 src/main.rs | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/main.rs b/src/main.rs
index 4846068..1ad9068 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -133,6 +133,8 @@ fn real_main() -> i32 {
         for ext_issue in req {
             let issue = repo.new_named_issue(format!("github-issue-{}", ext_issue.number)).unwrap();
 
+            progress_bar.message(&format!("[ #{} {} ] ", ext_issue.number, ext_issue.title));
+
             issue.new_record(Files::from(vec![(".type/SummaryChanged", &b""[..]),
                                               (".timestamp", ext_issue.created_at.as_bytes()),
                                               ("text", ext_issue.title.as_bytes()),
@@ -205,6 +207,7 @@ fn real_main() -> i32 {
         for pr in preq {
 
             let issue = repo.new_named_issue(format!("github-pr-{}", pr.number)).unwrap();
+            progress_bar.message(&format!("[ #{} {} ] ", pr.number, pr.title));
 
             issue.new_record(Files::from(vec![(".type/SummaryChanged", &b""[..]),
                                   (".timestamp", pr.created_at.as_bytes()),
-- 
2.15.0

